nuggetsapp/models.py

```diff
@@ -59,5 +66,13 @@ def add_new_nugget_for_user(cls, nugget, user, is_owner=False):
 
     @classmethod
     def get_nuggets_by_user(cls, user):
-        return [x.nugget for x in cls.objects.filter(user = user)]
+        return [x.nugget for x in cls.objects.filter(Q(user = user) & Q(is_deleted = False))]
+                
+    @classmethod
+    def get_todays_review_nuggets_by_user(cls, user):
+        review_interval_days = [1, 3, 7, 14, 30, 90, 180, 360, 720]
+        review_dates = [date_for_days_from_today(n) for n in review_interval_days]
+        return [x.nugget for x in  cls.objects.filter(Q(user = user) & Q(is_deleted = False) & Q(created_at__date__in = review_dates))]
```

Comment:
I don't think we should filter by is_deleted by default.  This means that if we ever wanted to load up a deleted user's nuggets page, this method would return nothing.  To me the best approach is to pass in a default parameter as an override to this method, like disallow_deleted=False, then add some logic to add the Q(is_deleted=False) filter when this method is explicitly called with disallow_deleted=True.